mail_outgoing_server_by_model/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,3 @@
+Allow to select an outgoing mail server per model.
+
+To make it work, the model must inherit `mail.thread`.
```

Comment:
It seems to only be active for comment on the mail.thread but not when sending a report, would be good to mention that.